src/runtime/pythonengine.cs

```diff
@@ -406,15 +393,47 @@ public static PyObject ModuleFromString(string name, string code)
 
 
         /// <summary>
+        /// Eval Method
+        /// </summary>
+        /// <remarks>
+        /// Evaluate a Python expression and returns the result.
+        /// It's a subset of Python eval function.
+        /// </remarks>
+        public static PyObject Eval(string code, IntPtr? globals = null, IntPtr? locals = null)
+        {
+            PyObject result = RunString(code, globals, locals, RunFlagType.Eval);
+            return result;
+        }
+
+
+        /// <summary>
+        /// Exec Method
+        /// </summary>
+        /// <remarks>
+        /// Run a string containing Python code. 
+        /// It's a subset of Python exec function.
+        /// </remarks>
+        public static void Exec(string code, IntPtr? globals = null, IntPtr? locals = null)
+        {
+            PyObject result = RunString(code, globals, locals, RunFlagType.File);
+            if (result.obj != Runtime.PyNone)
+            {
+                throw new PythonException();
+            }
+            result.Dispose();
+        }
+
+
+        /// <summary>
         /// RunString Method
         /// </summary>
         /// <remarks>
         /// Run a string containing Python code. Returns the result of
         /// executing the code string as a PyObject instance, or null if
         /// an exception was raised.
         /// </remarks>
-        public static PyObject RunString(
-            string code, IntPtr? globals = null, IntPtr? locals = null
+        internal static PyObject RunString(
```

Comment:
@vmuriart I agree with deprecating this method first. When this method become internal, the type of its last argument _flag can be changed from enum RunFlagType to IntPtr, and the enum RunFlagType can be removed. In the Eval and Exec methods, it can calls this RunString method using Runtime.Py_eval_input and Runtime.Py_file_input for this argument.